### ISSUE TITLE
support new LTS 18.x ?

### ISSUE THREAD

**Issue by Soontao** [nodeshift/opossum#655]
Hi Maintainers, 

just a reminder, nodejs has released 18.x as the new LTS version, maybe this line could be updated. 

https://github.com/nodeshift/opossum/blob/0491f4899c13f700b3a1c7872f52d49fa8bcf517/package.json#L72

otherwise, npm will show warnings

```bash
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'opossum@6.3.0',
npm WARN EBADENGINE   required: { node: '^16 || ^14 || ^12' },
npm WARN EBADENGINE   current: { node: 'v18.3.0', npm: '8.11.0' }
npm WARN EBADENGINE }
```

Regards,
Theo

**Comment by lucy-xiao** [comment]
+1

**Comment by lholmquist** [comment]
Sorry i missed this,  i was at OpenJS world last week.

i'll update it asap

**Comment by richardlau** [comment]
> just a reminder, nodejs has released 18.x as the new LTS version, maybe this line could be updated.

FWIW Node.js 18 does not become LTS until October 2022.

**Comment by lholmquist** [comment]
@lucy-xiao yes,  i'll do that today

**Comment by lholmquist** [comment]
just released 6.4.0
